src/xdocs/checks/misc/descendanttoken.xml

```diff
@@ -36,10 +36,7 @@
             <tr>
               <td>limitedTokens</td>
               <td>Specify set of tokens with limited occurrences as descendants.</td>
-              <td>
-                subset of tokens
-                <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html">TokenTypes</a>
```

Comment:
Let's put this valuable link to actual type of string values to property/setter description `<td>Specify set of tokens .....` in this case users will know that it is same.

Reply:
@romani, i have made changes. please let me know if understood the requested changes whenever convenient for you.

Reply:
I am ok, with such update.
I can move conversation with @rnveach  to separate issue to discuss it separately and not blocking this migration.